### ISSUE TITLE
Dung & carrion incorrect

### ISSUE THREAD

**Issue by HDean** [BiologicalRecordsCentre/Pantheon#200]
In the latest release, dung and carrion do show in the "tabular - other" table and also in the hierarchy view but not correctly. Dung and carrion are separated into which biotope and habitat they come under.
Using pasted species list 660, which are just dung beetles, the "tabular - other" report shows 4 instances of "dung":
open habitats >> tall sward >> dung & carrion >> dung
open habitats >> short sward >> dung & carrion >> dung
dung & carrion >> dung
and then another open habitats >> tall sward >> dung & carrion >> dung which duplicates one species from the first instance of this part of the hierarchy.
Total number of species is 67 but because of the duplication in the tall sward, when you add the totals of the 4 instances you get 68.

![image](https://user-images.githubusercontent.com/17744201/33898075-3bac95a0-df5f-11e7-8010-f8fef47fb0ce.png)

What we would like to see is just:
dung & carrion >> dung 
and the total number of species = 67.

**Comment by HDean** [comment]
Hi @johnvanbreda , sorry, this issue doesn't seem to be going away!

**Comment by johnvanbreda** [comment]
This is now fixed. In testing this though, I found another issue. See http://www.brc.ac.uk/pantheon/habitats-resources/tabular-other?dynamic-conservation_group_id=&dynamic-sample_id=3139437&dynamic-sample_type=sample, level 2, shows 20 species for leaves and/or stems. But, if you click on this link, you get a table with 44 species. This is because the table shown is the table of all leaves and/or stems linked species in the list, whereas the 20 species show only those which are leaves and/or stems AND plant associated. So there are species which are associated with a subcategory without being associated with the parent category, causing this discrepency. An example is Anoplus plantaris. 
Is this correct? How should it behave, or is it a data error?
